src/amo/pages/AddonReviewList/index.js

```diff
@@ -195,6 +195,10 @@ export class AddonReviewListBase extends React.Component<InternalProps> {
     );
   }
 
+  generateRandomLoadingReviews(min: number, max: number) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  }
```

Comment:
Why do you need to generate a random number? I think let's keep 4 if there is no `addonReviewCount`. Is that even possible?

Reply:
Ah, I did not see this. We have a fixed number right now, right? I'd start with that first. We can still iterate to improve.

Reply:
Okay gotcha - and yeah we currently use a hard coded value of 4